index.bs

```diff
@@ -131,9 +131,26 @@ Device identification and classification (e.g. device type and class) based on a
 
 To reduce fingerprinting risk, reported value is rounded to single most significant bit, as opposed to reporting the exact value. In addition, an upper and lower bound is placed on the reported values.
 
+IANA considerations {#iana}
+===================
+This document defines the `Device-Memory` HTTP request header field, and register them in the permanent message header field registery ([[RFC3864]]).
+
+Device-Memory Header field {#iana-device-memory}
+---------------------------------
+Header field name: Device-Memory
+
+Applicable protocol: http
+
+Status: standard
+
+Author/Change controller: IETF
+
+Specification document: this specification ([[#sec-device-memory-client-hint-header]])
```

Comment:
Optional -- If this is going to be exported into the actual registry, then we can put `sec-device-memory-client-hint-header` into a `Required IDs` section in the top metadata, to ensure that the links don't break if the document changes.